jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java

```diff
@@ -95,47 +95,47 @@ public void counterFileInputTest() {
         counterFileInput.counter().has().text("2 files (62 B in total)");
     }
 
-    @Test
+    @Test(description = "Test checks that file input is dens or not")
```

Comment:
misprint 'densE' :)

Reply:
I wish this misprint should be 'dance'!
Fixed

Reply:
:) :) :)